package.json

```diff
@@ -95,8 +95,8 @@
     "pretender": "0.10.1",
     "qunitjs": "1.20.0",
     "rimraf": "2.5.4",
-    "rollup": "0.34.7",
-    "rollup-plugin-commonjs": "3.3.1",
+    "rollup": "~0.36.3",
+    "rollup-plugin-commonjs": "~5.0.5",
```

Comment:
I'm down with trying yarn for a while. I tried it on some of the other modules I'm working on and it's so nice.